test/appSession.customStore.tests.js

```diff
@@ -0,0 +1,156 @@
+const { promisify } = require('util');
+const assert = require('chai').assert;
+const request = require('request-promise-native').defaults({
+  simple: false,
+  resolveWithFullResponse: true,
+});
+
+const appSession = require('../lib/appSession');
+const { get: getConfig } = require('../lib/config');
+const { create: createServer } = require('./fixture/server');
+const redis = require('redis-mock');
+const RedisStore = require('connect-redis')({ Store: class Store {} });
```

Comment:
This is bit of a pain, most session stores expect the `express-session` instance to define the store interface

Reply:
This **_is_** a pretty awkward import.  We could alternatively have them pass in the connector (default export) with store options, and then expose then instantiated the store.